packages/product-feed-heureka/src/Model/FeedItem/HeurekaFeedItemFactory.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+namespace Shopsys\ProductFeed\HeurekaBundle\Model\FeedItem;
+
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
+use Shopsys\FrameworkBundle\Model\Category\CategoryFacade;
+use Shopsys\FrameworkBundle\Model\Pricing\Price;
+use Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculationForUser;
+use Shopsys\FrameworkBundle\Model\Product\Product;
+use Shopsys\ProductFeed\HeurekaBundle\Model\HeurekaCategory\HeurekaCategoryFacade;
+
+class HeurekaFeedItemFactory
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\Pricing\ProductPriceCalculationForUser
+     */
+    protected $productPriceCalculationForUser;
+
+    /**
+     * @var \Shopsys\ProductFeed\HeurekaBundle\Model\FeedItem\HeurekaProductDataBatchLoader
+     */
+    protected $productDataBatchLoader;
```

Comment:
OK, I'll try to avoid it in the future. For me the `heureka` prefix was not important from the point of view of this class, but I get your point. I'll leave it as-is if you don't object, OK?